src/plugins/auto_batch/auto_batch.hpp

```diff
@@ -106,6 +106,7 @@ class AutoBatchInferRequest : public InferenceEngine::IInferRequestInternal {
     void CopyOutputsIfNeeded();
     AutoBatchExecutableNetwork::WorkerInferRequest& _myBatchedRequestWrapper;
     std::exception_ptr _exceptionPtr;
+    bool _wasBatchedRequestUsed;
```

Comment:
`= false;`

Reply:
Why not true?
Both values  are incorrect as initial

Reply:
Just add default initialization.

Reply:
moved to enum to explicitly have un-initialized state